app/code/Magento/Bundle/Model/ResourceModel/Selection/Collection.php

```diff
@@ -215,7 +215,7 @@ public function joinPrices($websiteId)
     public function setOptionIdsFilter($optionIds)
     {
         if (!empty($optionIds)) {
-            $this->getSelect()->where('selection.option_id IN (?)', $optionIds);
+            $this->getSelect()->where('selection.option_id IN (?)', $optionIds, \Zend_Db::BIGINT_TYPE);
```

Comment:
Is there a reason to be using Zend_Db::BIGINT_TYPE instead of Zend_Db::INT_TYPE? I guess all fields that we are using on the where the condition is an integer.